### DISCUSSION TITLE
How to write jslt to achieve the effect of 【firstNotNull】

### DISCUSSION THREAD

**Discussion by ChengHoHang** [schibsted/jslt#289] [Q&A] [answered]
for example，here my jslt input
```
// objectB and objectC nonNull
{
    "objectA":null,
    "objectB": ...
    "objectC": ...
}
```

so how to write jslt template to achieve the effect of 【firstNotNull】instead of using **if objectA != null else if else if .....**

This allows me to call like this
```
let firstNonNullKey = firstNonNull(.objectA, .objectB, .objectC, .......)
```

am i missing something?  please give some help

**Comment by larsga** [comment, marked as answer]
The [fallback](https://github.com/schibsted/jslt/blob/master/functions.md#fallbackarg1-arg2----value) function does exactly what you're asking for.

**Reply by ChengHoHang** [reply]
thank you,  it works for me!
